stdplugins/sp_h.py

```diff
@@ -1,8 +1,6 @@
 from telethon import events
 from telethon.tl import types
 
-@borg.on(events.NewMessage(pattern=r"^h$", outgoing=False))
+@borg.on(events.NewMessage(pattern="^h$", outgoing=False))
```

Comment:
how to define **@borg** ?

Reply:
with care

Reply:
I dont understand can you explain

Reply:
yes, it was done rather carefully.


care (noun): A concerned or troubled state of mind, as that arising from serious responsibility; worry.

Reply:
@muhammedfurkan Stop being a skiddy and learn to code. It's obviously injected when the plugin is loaded.